datafusion/src/physical_plan/hash_aggregate.rs

```diff
@@ -610,10 +615,14 @@ struct Accumulators {
     map: RawTable<(u64, usize)>,
 
     /// State for each group
-    group_states: Vec<GroupState>,
+    //group_states: Vec<GroupState>,
+    group_states: BumpVec<'a, GroupState>,
 }
 
-impl std::fmt::Debug for Accumulators {
+// will not be used multiple async, so Accumulators is Send
+unsafe impl Send for Accumulators<'_> {}
```

Comment:
If it's not used why does it need to be `Send`? I personally prefer no unsafe, does this work without it?

Reply:
This doesn't work. because accumulators holds `bump` for use in async func. async func requires `send`, but bump is `ref` not `send`.